pgmuvi/lightcurve.py

```diff
@@ -328,6 +328,61 @@ def __init__(self, xdata, ydata, yerr=None,
         self.__FITTED_MAP = False
         self.__FITTED_MCMC = False
 
+    @classmethod
+    def from_table(cls, tab, format='votable', xcol='x', ycol='y', yerrcol='yerr', **kwargs):
+        """Instantiate a Lightcurve object with
+        data read in from a VOTable.
+
+        Parameters
+        ----------
+        tab: astropy.table.Table object or str or pathlib.Path instance
+            Table containing the input data. If str, name (with extension) of file
+            containing the input data. In this case, the format keyword must be set
+            accordingly.
+        format: str
+            Format of file containing input data. Must be a format supported by Table.read.
+            Only required if type(tab) is str.
+        xcol: str
+            Name of column in table that contains the x data
+        ycol: str
+            Name of column in table that contains the y data
+        yerrcol: str
+            Name of column in table that contains the yerr data
+        kwargs:
+            Arguments to be passed to the Lightcurve call
+        
+        Returns
+        ----------
+        Lightcurve object
+        """
+        from pathlib import Path
+        from astropy.table import Table
+        if isinstance(tab, str) or isinstance(tab, Path):
+            data = Table.read(str(tab), format=format)
+        elif isinstance(tab, astropy.table.Table):
+            pass
```

Comment:
should this line be:

```suggestion
            data = tab
```

?

Reply:
Fixed.